### PR TITLE
Should not be able to close asset using clawback 

### LINKED ISSUE #311: Should not be able to close asset using clawback
An asset clawback transaction with a `closeRemainderTo` payFlag should throw an error (which is the behaviour of algorand network). Clawbacks cannot close assets from an address - only the actual owner can.

### PR DESCRIPTION
**Pull request by ratik21**
- Should not be able to close asset using clawback, added test
- Also checked that asset transfer is only account if both asset sender and receiver are not `frozen`, but if amount is 0 then txn is accepted. Added test

closes #311

### CODE REVIEW

File: packages/runtime/src/errors/errors-list.ts [hunk 1]
```diff
@@ -385,6 +385,12 @@ const runtimeAsaErrors = {
     message: "All of the created assets should be in creator's account",
     title: "Asset Total Error",
     description: "Asset Total Error"
+  },
+  CANNOT_CLOSE_ASSET_BY_CLAWBACK: {
+    number: 1509,
+    message: `Assetholding of account cannot be closed by clawback`,
+    title: "Close asset by clawback error",
+    description: "Cannot close asset by clawback"
```
Comment (robert-zaremba) [inline packages/runtime/src/errors/errors-list.ts:393]: please add more details into description. Maybe a link to documentation.

**Review by robert-zaremba — APPROVED** [review]
pre-approving

File: packages/runtime/src/errors/errors-list.ts (hunk 1 above)
Comment (ratik21) [reply packages/runtime/src/errors/errors-list.ts:393]: I couldn't find in documentation regarding this behaviour. I got to know it by asking on discord. In documentation they mention `AssetCloseTo` param in clawback transaction reference - [https://developer.algorand.org/docs/reference/transactions/?query=clawback#asset-clawback-transaction](https://developer.algorand.org/docs/reference/transactions/?query=clawback#asset-clawback-transaction), which led to confusion

Comment (ratik21) [reply packages/runtime/src/errors/errors-list.ts:393]: I will add a link for general asset clawback behaviour

**Pull request merged**